src/helpers/resizedImages.ts

```diff
@@ -0,0 +1,29 @@
+import ResizedImages from '../models/ResizedImages'
+
```

Comment:
Let's call the file `resizedImagesCache`

Reply:
Resolved in [805ba34](https://github.com/BigWhaleLabs/obss-ipfs-uploader/pull/12/commits/805ba34f85b84583f8ed8b300a59a92e72b37ce6).